README.md

```diff
@@ -108,38 +110,36 @@ Use the `cdk` command-line toolkit to interact with your project:
  * `cdk synth`: synthesizes an AWS CloudFormation template for your app
  * `cdk diff`: compares your app with the deployed stack
 
-For a detailed walkthrough, see the [tutorial] in the AWS CDK [Developer Guide].
-
 ## Getting Help
 
-Please use these community resources for getting help. We use the GitHub issues
-for tracking bugs and feature requests.
+The best way to interact with our team is through GitHub. You can open an [issue](https://github.com/aws/aws-cdk/issues/new/choose) and choose from one of our templates for bug reports, feature requests, documentation issues, or guidance.
+
+If you have a support plan with AWS Support, you can also create a new [support case](https://console.aws.amazon.com/support/home#/).
 
+You may also find help on these community resources:
+* Look through the [API Reference](https://docs.aws.amazon.com/cdk/api/latest/docs/aws-construct-library.html) or [Developer Guide](https://docs.aws.amazon.com/cdk/latest/guide)
 * Ask a question on [Stack Overflow](https://stackoverflow.com/questions/tagged/aws-cdk)
   and tag it with `aws-cdk`
 * Come join the AWS CDK community on [Gitter](https://gitter.im/awslabs/aws-cdk)
 * Talk in the CDK channel of the [AWS Developers Slack workspace](https://awsdevelopers.slack.com) (invite required)
-* Open a support ticket with [AWS Support](https://console.aws.amazon.com/support/home#/)
-* If it turns out that you may have found a bug,
-  please open an [issue](https://github.com/aws/aws-cdk/issues/new)
+
+### Roadmap
+
+The [AWS CDK Roadmap project board](https://github.com/orgs/aws/projects/7) lets developers know about our upcoming features and priorities to help them plan how to best leverage the CDK and identify opportunities to contribute to the project. See [ROADMAP.md](https://github.com/aws/aws-cdk/blob/master/ROADMAP.md) for more information and FAQs.
 
 ## Contributing
 
 We welcome community contributions and pull requests. See
-[CONTRIBUTING](./CONTRIBUTING.md) for information on how to set up a development
+[CONTRIBUTING.md](./CONTRIBUTING.md) for information on how to set up a development
 environment and submit code.
 
-## Roadmap
-
-The [AWS CDK Roadmap project board] lets developers know about our upcoming features and priorities to help them plan how to best leverage the CDK and identify opportunities to contribute to the project. See [ROADMAP] for more information and FAQs.
-
-[AWS CDK Roadmap project board]: https://github.com/orgs/aws/projects/7
-[Roadmap]: (https://github.com/aws/aws-cdk/ROADMAP.md)
-
-## License
```

Comment:
I think the license needs to be explicitly stated in the README, no?

Reply:
I looked through the Open Source guidelines and couldn't find info on this. Where did you see that guideline defined?

Reply:
Not sure it's a guideline, but it's a very very common practice in the open source world I think. I am okay to omit, but just wanted to bring up.

Reply:
Brought is up and the OSDS team decided against keeping it in because it's already mentioned in the "About" section of the home page